src/commands/capes.js

```diff
@@ -0,0 +1,105 @@
+                    type: 3,
+                    required: true
+                },
+                /*{
+                    name: "cape",
+                    description: "Cape",
+                    type: 3,
+                    required: true,
+                    choices: [
+                        {
+                            name: "Contributor",
+                            name: "Contributor cape",
+                            value: ""
+                        },
+                    ]
+                }*/ //anal sex
+            ]
+        }
+    ],
+
+    run: async (interaction) => {
+
+        for (let index of interaction.options.data) {
+            switch (index.name) {
+                case "pull": {
+                    const isForced = interaction.options.getBoolean("force") ?? false
+                    const pullResult = await capeUtils.pull(isForced);
+                    if (!pullResult && !isForced) return interaction.reply({ embeds: [embedUtils.error('Add `force:true` in the options to override local data.')] });
+                    return interaction.reply({ embeds: [embedUtils.success("Pulled!")] });
+                }
+                case "push": {
+                    const pushResult = await capeUtils.push();
+                    if (!pushResult) { await capeUtils.pull(); await capeUtils.push(); }
+                    return interaction.reply({ embeds: [embedUtils.success('Pushed to remote.')] })
+                }
+                case "add": {
+                    const minecraft_username = await interaction.options.getString("minecraft_username")
+                    const user = await interaction.options.getInteger("user_id")
+                    try {
+                        const _ = await axios.get(`https://discordapp.com/api/v9/users/${user}`, {
+                            headers: {
+                                authorization: "Bot " + process.env.TOKEN
+                            }
+                        })
+                    } catch {
+                        return await interaction.reply({ embeds: [embedUtils.error("Invalid user")] })
+                    }
+                    const minecraft_uuid = await uuidUtils.usernameToUUID(minecraft_username);
+                    console.log(minecraft_uuid)
+                    if (!minecraft_uuid) return await interaction.reply({ embeds: [embedUtils.error("Invalid username or nonexistent player")] })
+                    const addResult = capeUtils.add(user, minecraft_uuid);
+                    if (!addResult) return interaction.reply({ embeds: [embedUtils.error('No local data found. Please pull first.')] })
+                    return interaction.reply({ embeds: [embedUtils.success(`Added <@${user}>, Info: \`\`\`Username: ${minecraft_username}\nUUID: ${minecraft_uuid}\`\`\``)] })
+                }
+                default: return interaction.reply({ embeds: [embedUtils.error('Why you choose nothing :skull:')] })
```

Comment:
https://primer.style/design/foundations/content

Reply:
You can't even access the default one
`for (let index of interaction.options.data) {}`

Reply:
Fixed